tests/models/test_taskinstance.py

```diff
@@ -2856,8 +2856,9 @@ def on_execute_callable(context):
         ],
     )
     @patch("logging.Logger.exception")
+    @patch("logging.Logger.info")
```

Comment:
[`caplog` fixture](https://docs.pytest.org/en/stable/how-to/logging.html#caplog-fixture)

Reply:
I guess it could simplify test case and do not required to repeat same actions from the callback itself.

```python

def test_finished_callbacks_handle_and_log_exception(self, finished_state, create_task_instance, caplog):
    ...
    for callback_input in [[on_finish_callable], on_finish_callable]:
        caplog.clear()
        _run_finished_callback(callbacks=callback_input, context={})

        assert called
        assert not completed
        assert "Executing on_finish_callable callback" in caplog.text
        assert "Error when executing on_finish_callable callback" in caplog.text

```

Reply:
And in general this test case have some drawbacks. 
E.g. `called` set only in the beginning to False, after that callback executed twice in the loop.
So assign `called = completed = False` also should be call into the beginning of the loop,

Reply:
Thanks for notice it, I changed it.